src/const_eval.md

```diff
@@ -23,7 +23,10 @@ to be run.
 * [Const parameters].
 * [Paths] to [functions] and [constants].
   Recursively defining constants is not allowed.
-* Paths to [statics]. These are only allowed within the initializer of a static.
+* Paths to immutable [statics] with these restrictions and observations.
```

Comment:
I think this might still need to be updated, since it allows `static mut`.

```suggestion
* Paths to [statics] with these restrictions and observations:
```

Reply:
Applied